### PR TITLE
31986: Update UPS table to match the new field functionality.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/services.rst [hunk 1]
```diff
@@ -1955,13 +1955,12 @@ UPS Configuration screen.
    |                               |                | for more details about configuring multiple systems with a single UPS.                                                 |
    |                               |                |                                                                                                                        |
    +-------------------------------+----------------+------------------------------------------------------------------------------------------------------------------------+
-   | Port / Hostname / Remote Port | drop-down menu | :guilabel:`Port`: Enter the serial or USB port the UPS is connected to (see :ref:`NOTE <UPS USB>`).                    |
+   | Port or Hostname              | drop-down menu | Enter the serial or USB port the UPS is using (see :ref:`NOTE <UPS USB>`).                                             |
```
Comment (wblock) [inline userguide/services.rst]: "is using" is kind of ambiguous.  Not clear on how it is using it.  Better is to be specific, but let's still avoid that split infinitive:
"Enter the serial or USB port connected to the UPS (see..."

**Comment by dlavigne** [comment]
This contains text improvements that also need to be ported to master.

**Comment by a-stjohn** [comment]
> This contains text improvements that also need to be ported to master.

Hey, Dru. Just checked the master branch and the text improvements are present.

**Comment by dlavigne** [comment]
Yup, thanks to legacy doc commit: https://github.com/freenas/freenas-docs/commit/c96f97e85169a7b1c8ee724783e77171ee556f2d :-)

**Pull request merged**